src/Microsoft.ServiceFabric.Data.Indexing.Persistent/FilterableIndex.cs

```diff
@@ -1,12 +1,20 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Linq.Expressions;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.ServiceFabric.Data.Collections;
 
 namespace Microsoft.ServiceFabric.Data.Indexing.Persistent
 {
+
+    public enum RangeFilterType
+    {
+        INCLUSIVE,
+        EXCLUSIVE
```

Comment:
(nit): can you make these not all caps?  Enums in C# are typically just proper nouns, so `Inclusive` and `Exclusive`

Reply:
fixed